packages/fluentui/react-northstar/src/utils/felaRenderer.tsx

```diff
@@ -12,6 +12,49 @@ import felaInvokeKeyframesPlugin from './felaInvokeKeyframesPlugin';
 import felaSanitizeCss from './felaSanitizeCssPlugin';
 import felaStylisEnhancer from './felaStylisEnhancer';
 
+import cssifyDeclaration from 'css-in-js-utils/lib/cssifyDeclaration';
+import {
+  // @ts-ignore
+  generateCombinedMediaQuery,
+  // @ts-ignore
+  generateCSSSelector,
+  // @ts-ignore
+  isMediaQuery,
+  // @ts-ignore
+  isNestedSelector,
+  // @ts-ignore
+  isSupport,
+  // @ts-ignore
+  isUndefinedValue,
+  // @ts-ignore
+  normalizeNestedProperty,
+  RULE_TYPE,
+} from 'fela-utils';
+
+function isPlainObject(val) {
+  return val != null && typeof val === 'object' && Array.isArray(val) === false;
+}
+
+function camelCasePropertyNonRegex(property) {
```

Comment:
A single change in this PR, replaces https://github.com/robinweser/css-in-js-utils/blob/master/modules/camelCaseProperty.js